gips/inventory/dbinv/models.py

```diff
@@ -1,6 +1,22 @@
 from __future__ import unicode_literals
 
-from django.db import models
+from django.contrib.gis.db import models
+from django.contrib.postgres.fields import HStoreField
```

Comment:
I have no idea what's going on, but the line 'from django.contrib.postgres.fields import HStoreField' does require Postgres